lib/intervention/client.rb

```diff
@@ -1,34 +1,47 @@
 module Intervention
   class Client < EventMachine::Connection
-    attr_reader :server, :parser
+    attr_reader :server, :parser, :enable_tls
+
+    def initialize(config = nil)
+      @enable_tls = !config.nil?
```

Comment:
enable tls if there is a config?
This seems a bit odd, should this not be something like `= config.tls`?